.github/workflows/release.yaml

```diff
@@ -153,9 +155,18 @@ jobs:
 
     # Push the Docker image as part of the release pipeline.
     - name: Push the Docker image
+      id: push-docker
       env:
         RELEASE_TAG: ${{ steps.release-info.outputs.release-tag }}
-      run: make push-docker
+      run: |
+        make push-docker
+        IMAGE_AND_DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' "$IMAGE_NAME")
```

Comment:
When I try to build the Docker image (to create an image called `ghcr.io/oracle-samples/macaron
`) and run this script manually, it cannot resolve. This is the error that I have:
```
docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/oracle-samples/macaron

template parsing error: template: :1:2: executing "" at <index .RepoDigests 0>: error calling index: reflect: slice index out of range
```
However, if I run this script with an image that currently exists in ghcr (i.e `ghcr.io/oracle-samples/macaron-base
`, it would work normally.
It is correct to understand that `make push-docker` will push the image to ghcr previously and this script will work fine in that case?

Reply:
> It is correct to understand that `make push-docker` will push the image to ghcr previously and this script will work fine in that case?

Yes, that's right. Certain information, such as `RepoDigests` is not finalized until the image is pushed. That's why you get an empty field before pushing the image.